lib/services/bitcoind.js

```diff
@@ -69,6 +71,14 @@ Bitcoin.prototype._loadConfiguration = function() {
       'Please add "txindex=1" to your configuration and reindex an existing database if ' +
       'necessary with reindex=1'
   );
+
+  if (this.configuration.reindex && this.configuration.reindex === 1) {
+    log.warn('Reindex option is currently enabled. This means that bitcoind is undergoing a reindex. ' +
+      'Once the reindex is complete, the rest of bitcore-node services will start. ' +
+      'WARNING!!! Be sure to remove \'reindex=1\' from your bitcoin.conf -BEFORE- restarting bitcore-node.');
```

Comment:
Could be better to mention that the reindex flag will start the index from beginning every time the node is started, so it should be removed after the reindex has be initiated.